frontend-react/e2e/pages/authenticated/admin/organization-receiver-message-test.ts

```diff
@@ -0,0 +1,121 @@
+import type { Locator } from "@playwright/test";
+import {
+    errorMessageResult,
+    passMessageResult,
+} from "../../../../src/components/Admin/MessageTesting/MessageTestingResult.fixtures";
+import { RSMessage } from "../../../../src/config/endpoints/reports";
+import { MOCK_GET_TEST_MESSAGES } from "../../../mocks/message-test";
+import { BasePage, BasePageTestArgs, type RouteHandlerFulfillEntry } from "../../BasePage";
+
+/**
+ * Uses org ignore's FULL_ELR receiver
+ */
+export class OrganizationReceiverMessageTestPage extends BasePage {
+    static readonly API_REPORTS_TESTING = "/api/reports/testing";
+    static readonly API_REPORTS_TEST = "/api/reports/testing/test?*";
+    protected customI: number;
+    testMessages: RSMessage[];
+
+    readonly expectedStatusSuccess = /^Test passed/;
```

Comment:
This file exists seems dubious at best _but_ it exists and we use it, so maybe we can reference these strings straight from here: https://github.com/CDCgov/prime-reportstream/blob/main/frontend-react/src/components/Admin/MessageTesting/language.json

Reply:
It was an idea similar to when I started making unit test pieces like fixtures separated for reuse in testing ecosystem (such as E2E). The idea must've worked because you immediately thought to use it in this case 😄.